.circleci/config.yml

```diff
@@ -331,6 +370,36 @@ jobs:
               $host.SetShouldExit(1)
             }
 
+  merge-release-branch:
+    parameters:
+      e:
+        type: executor
+    executor: << parameters.e >>
+    steps:
+      - slack/notify:
+          channel: 'pdt_releases'
+          color: '#FFDEAD'
+          message: 'Merging release branch into main'
+      - ssh-config
+      - checkout
+      - gh-config
+      - prepare-release-environment-variables
+      - merge-validation
+      - run:
+          name: 'Rebasing main off of release branch'
+          command: |
+            echo "Rebasing main off of release branch ${RELEASE_VERSION}"
+            git checkout main
+            git rebase -Xtheirs release/v${RELEASE_VERSION}
```

Comment:
what does `-Xtheirs` do here?

Reply:
In the event of a conflict, -Xthiers should take the changes from the incoming branch (in this case the release branch) to override the conflict automatically.